spec/spec_helper.rb

```diff
@@ -103,6 +103,7 @@ class TestApp < Rails::Application
 
   patch :update_with_error_on_base, to: 'posts#update_with_error_on_base'
 
+  get :get_related_resources, to: 'posts#get_related_resources'
```

Comment:
I tried change the routes this way but it didn't work for me. I don't know what I did wrong.
But yes this would be a better configuration for the tests.